CHANGES.txt

```diff
@@ -27,6 +27,8 @@ Changes
 Fixes
 =====
 
+- Use only primary shards to calculate table stats.
```

Comment:
I think we should be more descriptive on what issue are we fixing here (I guess it's about reflecting more accurate stats - num docs etc)

Reply:
Addressed your concern.